transitions/RotateScaleVanish.glsl

```diff
@@ -0,0 +1,40 @@
+// Author: Mark Craig
+// mrmcsoftware on github and youtube ( http://www.youtube.com/MrMcSoftware )
+// License: MIT
+
+// RotateScaleVanish Transition by Mark Craig (Copyright © 2022)
+
+uniform bool FadeInSecond; // = true
+uniform bool ReverseEffect; // = false
+uniform bool ReverseRotation; // = false
+
+#define M_PI 3.14159265358979323846
+#define _TWOPI 6.283185307179586476925286766559
+
+vec4 transition(vec2 uv)
+{
+if (progress == 0.0) { return(getFromColor(uv)); }
+else if (progress == 1.0) { return(getToColor(uv)); }
+vec2 iResolution = vec2(ratio, 1.0);
+float t;
+if (ReverseEffect) { t = 1.0 - progress; } else { t = progress; }
+float theta = ReverseRotation ? _TWOPI * t : -_TWOPI * t;
+float c1 = cos(theta);
+float s1 = sin(theta);
+float rad = 1.0 - t;
+if (rad <= 0.0) { rad = 0.00001; }
```

Comment:
sounds like you can instead `rad = max(0.00001, 1.0 - t)`

Reply:
Done. Since I don't know a GPU's internal instruction set, I didn't know if max would translate to a special instruction that would then be faster/better than an if, but at worst, max wouldn't be worse, so it's a good idea. Thanks for the suggestion.